projects/lib-magnetar-quill/src/lib/components/editor/editor.component.ts

```
@@ -174,29 +625,63 @@ export class EditorComponent implements OnInit, AfterViewInit, OnChanges, DoChec
+ * Maintenance 08: method `EditorComponent`.`sanitizeHtml()` should be updated together with its surrounding call sites, tests, templates, and lifecycle wiring whenever the implementation intent or observable behavior changes.
+ * How 09: method `EditorComponent`.`sanitizeHtml()` is executed through concrete statements in the implementation body, and this line records that the algorithmic path and state transitions are considered part of the documented design.
+ */
+private sanitizeHtml(html: string): string {
+    const template = document.createElement('template');
+    template.innerHTML = html;
 
-    // Remove all inline styles
-    html = html.replace(/style="[^"]*"/gi, '');
+    template.content
+      .querySelectorAll('meta, script, iframe, embed, object')
+      .forEach(element => element.remove());
```

Biome (2.5.0)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)